linux-support.md

```diff
@@ -1,38 +1,41 @@
 # .NET Support and Compatibility for Linux Distributions
 
-.NET is [supported by Microsoft](os-lifecycle-policy.md) and by [various commercial vendors and the community](linux.md) on multiple Linux distributions.
+.NET is supported by multiple [commercial and community organizations](support.md) on [multiple Linux distributions](linux.md).
 
-.NET can typically be run on any Linux distribution, via:
+.NET can be run on any Linux distribution, via:
```

Comment:
can we say we run on any Linux distro?

Reply:
I'm unaware of any where that fails. .NET doesn't run well on FreeBSD, but that's not Linux.